pennylane/operation.py

```diff
@@ -1511,6 +1511,13 @@ def __matmul__(self, other):
 
     def __sub__(self, other):
         """The subtraction operation of Operator-Operator objects and Operator-scalar."""
+        if active_new_opmath():
```

Comment:
If this is in `Operator`, do we need the `active_new_opmath` toggle? It think we could make this the default for all cases.

Reply:
tbh, the whole thing is so crazy that I'm fine with anything. Eg:
```pycon
>>> op = qml.S(0) - qml.PauliX(1)
>>> op
S(wires=[0]) +   (-1) [X1]
>>> type(op)
<class 'pennylane.ops.op_math.sum.Sum'>
>>> type(op[1])
<class 'pennylane.ops.qubit.hamiltonian.Hamiltonian'>
```
^ happens on master _and_ this branch. I'll make the change